### ISSUE TITLE
Full Stack Dev / Team Lead in Brisbane CBD ($120k - $140k)

### ISSUE THREAD

**Issue by Camm0** [BrisJS/meetups#234]
**Who we are**
We are Touch Panel Control a UK based company starting up a development arm in Brisbane. We develop across multiple platforms with apps for Home Automation, Telemedicine, and Videoconferencing. 

As a dynamic small group in Brisbane passionate about home automation, corporate huddle space analytics, room / environment control in ProAv, we are looking to grow our Brisbane arm with a Lead JS developer who can take charge of the project, bring creative direction to our architecture and build a team that can help deliver the vision. 

**Basics on Position**
Ideally, looking for:
- Team leader/lead developer – experienced in building and managing teams of 5-6 for at least 2-3 years (including offshore).
- Be part of the project scope, bring your creative architectural ideas
- Build a team with the right skillset
- Some exposure to basic BA skills, scoping requirements from non-technical people would be great.
- The desire to teach others and help them grow in areas that will benefit them and the team
- Full stack Dev, up to date with the latest JS frameworks and trends. (Most projects we use are Angular, but open to what suits your skillset)
- Basically, you know your stuff and you're good at what you do (not just starting out)

We are a fun bunch of people and believe this is a great opportunity for a Team Lead to grow to the next level.

**Contact Me**
If you're interested, contact me camran@touchpanelcontrol.com - Keen to move fast on this.
